overview.md

```diff
@@ -0,0 +1,180 @@
+space and time. Examples include Imagery (from satellites, planes and drones), SAR, Point Clouds (from
+LiDAR, Structure from Motion, etc), Data Cubes, Full Motion Video, and data derived from any of those.
+The key is that the GeoJSON is not the actual 'thing', but instead references files and serves as an
+index to the 'assets'. It is [not recommended](best-practices.md#representing-vector-layers-in-stac) 
+to use STAC to refer to traditional vector data layers (shapefile, geopackage) as assets, as they
+don't quite fit conceptually. 
+
+## Catalogs vs Collections
+
+Before we go deep into the Catalogs and Collections, it is worth explaining the relationship 
+between the two and when you might want to use one or the other. 
+
+A Catalog is a very simple construct - it just provides links to Items or to other Catalogs. 
+The closest analog is a folder in a file structure, it is the container for Items, but it can 
+also hold other containers (folders / catalogs). 
+
+The Collection specification extends the Catalog spec, so every Collection can serve as a catalog, 
+requiring the same fields. But it has a number of additional fields: license, extent (spatial and 
+temporal), providers, keywords and summaries. So every Item in the Collection must share those fields, 
+and if Items are in a Collection they must link back to their Collection, so clients can easily find 
+fields like the license. 
+
+But what *should* go in a Collection, versus just in a Catalog?  A collection will generally consist of
+a set of assets that are defined with the same properties and share higher level metadata. In the 
+satellite world these would typically all come from the same sensor or constellation. It corresponds
+directly to what others call a 'dataset series' (ESA, ISO 19115), 'collection' (CNES, NASA), and 
+'dataset (JAXA, DCAT)'. So if all your items have the same properties, they probably belong in 
+the same Collection. But the construct is deliberately flexible, as there may be good reasons
+to 'break' the recommendation. 
+
+Catalogs in turn are used for two main things:
+
+* Split overly large collections into groups
+* Group collections into a catalog of collections (i.e. as entry point).
+
+The first case allows users to browse down into the items of big collections. A collection like
+Landsat usually would start with path and row Catalogs to group by geography, and then year, 
+month and day groups to enable deeper. [Dynamic catalogs](best-practices.md#dynamic-catalogs) can
+provide multiple grouping paths, serving as a sort of faceted search.
+
+The second case is used when one wants to represent diverse data in a single place. If an organization
+has an internal catalog with Landsat 8, Sentinel 2, NAIP data and several commercial imagery providers
+then they'd have a root catalog that would link to a number of different collections. 
+
+So in conclusion it's best to use collections for what you want user to find as starting point, and then
```

Comment:
Maybe add that users may also want to search on collections at some point...

Reply:
I think I thought about that, but then feared getting into all the details of that (like the role of ogc api - records in it, which I *hope* is our solution for collection-level search, but also realized we've not talked about that / have any implementations except pygeoapi). I can put a little call out to it.

Reply:
Put a little bit in here.